tests/modules/programs/firefox/profiles/bookmarks/expected-bookmarks-user.js

```diff
@@ -2,7 +2,7 @@
 
 
 
-user_pref("browser.bookmarks.file", "/nix/store/00000000000000000000000000000000-@name@-bookmarks.html");
+user_pref("browser.bookmarks.file", "/nix/store/00000000000000000000000000000000-bookmarks.html");
```

Comment:
The app name isn't actually necessary for this file, so I just removed it to make things simpler.